controllers/release/release_adapter.go

```diff
@@ -119,70 +122,71 @@ func (a *Adapter) EnsureFinalizerIsAdded() (reconciler.OperationResult, error) {
 // EnsureReleasePlanAdmissionEnabled is an operation that will ensure that the ReleasePlanAdmission is enabled.
 // If it is not, no further operations will occur for this Release.
 func (a *Adapter) EnsureReleasePlanAdmissionEnabled() (reconciler.OperationResult, error) {
-	_, err := a.getActiveReleasePlanAdmission()
+	_, err := a.loader.GetActiveReleasePlanAdmissionFromRelease(a.ctx, a.client, a.release)
```

Comment:
I don't think you should do this `FromRelease`.. it is the first operation, so it won't be there the first time we reconcile a release, right?